### DISCUSSION TITLE
⛔ 🛟 Support Request: I just need a question answered.

### DISCUSSION THREAD

**Discussion by mevo-0420** [elementor/elementor#23210] [Developers Community]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I have an elementary Pro license I have spoken with Elementor and they have passed me to four different Representatives with Elementor and none of them have answered my question. It has been a week now since I've asked the question.

And I cannot find any information about this online, or in forums.

### Describe the solution you'd like

I have Elementor Pro. I reached out to them through Elementary’s website to ask two questions. One was about connected accounts to Elementor’s licenses. It took three representatives to get that question finally answered. I have to mention that all three of these Representatives never even acknowledged the second question which was the more important of the two questions I was contacting them for.

I mentioned that nobody has responded to my question and they passed me on to a fourth representative. None of the representatives that I spoke to, has answered my question at all. 

Here is my question. Can anyone please help me with this?

Specs:
WordPress 6.2.2
ELementor & Pro: 3.14.1
Theme: Hello
Child Theme: Hello
Plugins (10 Total): WP Mail SMTP, White Label CMS, LiteSpeed Cache, Envato Elements, Elementor, Elementor Pro, Code Snippets, Yoast SEO, ELement Pack Pro (minimal use)

During my learning journey, I’ve made extensive modifications to the site, including creating and deleting multiple elements, installing and uninstalling plugins, and generally experimenting with various configurations. My concern is that these actions, while part of my learning process, may have inadvertently introduced unnecessary residual data and code into my database. This could include potential extra DOM elements, unnecessary code, bloat, surplus database tables, transient options, and other remnants of my exploratory process. Even though the site appears to be functioning properly, I worry that I may have unintentionally compromised its optimal performance or left it with unnecessary clutter. I also kind of made mistakes like putting custom CSS multiple times in multiple widgets all over the website, when I could have put it in Elementors custom CSS section, themes CSS Section, or even better the style sheet, and it would have worked globally instead of me duplicating the same CSS multiple times. And lots of other issues like that that I want to clean up.

I believe in doing things right, so I want to ensure that I’m launching the cleanest, most efficient site possible.

So, I’ve come up with a plan to address these concerns and I’d appreciate your input on it.

1. Set up a fresh staging site with WordPress and Elementor Pro installed, including the necessary plugins and settings
that mirrors my current site.

2. Optimize all images that will be used on the website and have them ready to go.

3. Begin working on one webpage at a time, starting with the first section on each page. Swap out existing images for the optimized versions, and fix any font size issues that were caused by switching from Elementor's default fonts to my hosted ones.

4. Check the responsiveness of each section on real devices to ensure they look good across all platforms.

5. After completing the adjustments on each page, run optimization plugins to clean up any potential bloat, improve performance, and to try reduce unnecessary residuals that might or could be saved out with templates

6. As I finish steps 3 - 5, I will start saving sections or the entire page as templates and export them to the new staging site.
    
At this point, before transferring templates, I will set up CSS properly, either in the Hello Themes Additional CSS, but more than likely I will put the CSS in the CSS Style sheet.

I believe this plan will help ensure that the website I launch is free of unnecessary code and data, and is as efficient and streamlined as possible. However, I would greatly appreciate your expert advice on this matter.

Question 1: Are there any potential issues I may have overlooked with this plan?

Question 2: Are there additional steps you would recommend to further improve the optimization process?

Thank you in advance for your time and assistance. I look forward to your reply.

**Comment by nicholaszein** [comment]
Hello @mevo-0420,

Thanks for contacting us! 🙏 

### 🛟 This is more of a support question, not really something we can help with here. 

👉 Please remember to check the [Help Center](https://elementor.com/help) or contact [Elementor Support](https://elementor.com/contact) before submitting an issue.


### ⚠️ Note that GitHub discussions should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards
